student.rb

```diff
@@ -5,11 +5,16 @@ class Student < Person
   def initialize(age, name, classroom, parent_permission)
     super(age, name)
     @id = id
-    @classroom = classroom
+    @classroom = Classroom.new(classroom)
```

Comment:
- [ ] Good job implementing the `add_to_classroom` method, you're pushing the classroom in the method therefore this line of code is not necessary kindly consider removing the `Classroom.new(classroom)` and replace it with an empty array `[]`

```suggestion
    @classroom = []
```